objects/ohitbox/Step_2.gml

```diff
@@ -22,228 +22,400 @@ if (!global.game_paused)
+					
+					collision_list[| i].owner.prevState = eState.BLOCKING;
+					collision_list[| i].owner.state = eState.HITSTOP; // Set the player's state to hitstop
 				
-				var particle = instance_create_layer(collision_list[| i].owner.x, collision_list[| i].owner.y, "Particles", oParticles);
-				with (particle) 
-				{
-					lifetime = 10;
-					sprite_index = sBlockEffect;
-					image_xscale = collision_list[| i].owner.image_xscale;
-				}
-			}
-			//Hitting	
-		else if attackProperty.attackType[hitboxID] != eAttackType.GRAB { 
-				if (collision_list[| i].owner.state != eState.BEING_GRABBED) 
-				{
-					collision_list[| i].owner.sprite_index = collision_list[| i].owner.CharacterSprites.hurt_Sprite;
-				} 
+					owner.prevState = owner.state; // Set the owner's previous state
+					owner.state = eState.HITSTOP;
 				
-				// Set the correct Sprite
-				if (!collision_list[| i].owner.isDestructibleObject) // Check if the hurbox is attatched to a destructible object
-				{
-					collision_list[| i].owner.prevState = eState.HURT; // Set the victim's previous state to HURT
-					collision_list[| i].owner.isShortHopping = false; // Make sure the victim is not using their shorthop fall speed.
+					// Handle if the opponent is Crouch blocking or not
+					if (collision_list[| i].owner.verticalMoveDir == -1)
+					{
+						collision_list[| i].owner.sprite_index = sRussel_Crouch_Block;
+						collision_list[| i].owner.isCrouchBlocking = true;
+					}
+					else 
+					{
+						collision_list[| i].owner.sprite_index = sRussel_Block;
+						collision_list[| i].owner.isCrouchBlocking = false;
+					}
+				
+					// Meter Build - P1 gets 75% meter, P2 gets 50%
+					collision_list[| i].owner.superMeter += floor(attackProperty.meterGain[hitboxID] * 0.5);
+					owner.superMeter += floor(attackProperty.meterGain[hitboxID] * 0.75);
+				
+					collision_list[| i].owner.knockbackVel = attackProperty.knockback[hitboxID];
+					owner.pushbackVel = attackProperty.pushback[hitboxID];
+					hasHit = true;
+					collision_list[| i].owner.blockstun = attackProperty.blockstun[hitboxID];
+					collision_list[| i].owner.shuffle = 0;
+					global.hitstop = attackProperty.attackHitstop[hitboxID];
+				
+					// Allow Cancelling
+					owner.cancelable = true;
+				
+					// Multiple hitboxes
+					ds_list_add(collision_list[| i].owner.hitByGroup, attackProperty.group[hitboxID]);
+				
+					// Depth Sorting
+					owner.depth = -1;
+					collision_list[| i].owner.depth = 0;
+				
+					// Reset Frame Advantage Counter
+					oGameManager.frameAdvantage = 0;
+				
+					var particle = instance_create_layer(collision_list[| i].owner.x, collision_list[| i].owner.y, "Particles", oParticles);
+					with (particle) 
+					{
+						lifetime = 10;
+						sprite_index = sBlockEffect;
+						image_xscale = collision_list[| i].owner.image_xscale;
+					}
 				}
+				//Hitting	
+			else if attackProperty.attackType[hitboxID] != eAttackType.GRAB { 

```

Comment:
The ```{``` needs to go on it's own line. Also the condition of a conditional should be enclosed in parentheses.